packages/client/package.json

```diff
@@ -15,6 +15,7 @@
   ],
   "devDependencies": {},
   "dependencies": {
```

Comment:
Can you bump the package version and publish a new version of the client?!

Reply:
I'll do that later when we deploy the webchat